portafly/portafly_api.yml

```
@@ -14,28 +14,103 @@ paths:
     description: A list of developer accounts.
     get:
       responses:
-        "200":
+        '200':
           $ref: '#/components/responses/developerAccountList'
       operationId: developerAccountList
       summary: Get developer account list
     parameters:
-    - name: page
-      description: Page in the paginated list. Defaults to 1.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: per_page
-      description: Number of results per page. Default and max is 500.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: state
-      description: Account state.
-      schema:
-        type: string
-      in: query
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: state
+        description: Account state.
+        schema:
+          type: string
+        in: query
+  /users:
+    summary: A list of user
+    description: A list of users
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/userList'
+      operationId: userList
+      summary: Get account users list
+    parameters:
+      - name: page
+        description: Current page for pagination
+        schema:
+          type: integer
+        in: query
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+  /accounts/invitations:
```

If it's for 1 account, then the ID should be in the URL:
```suggestion
  /accounts/:account_id/invitations:
```